tensorflow/api_template.__init__.py

```diff
@@ -27,7 +27,10 @@
 """
 # pylint: disable=g-bad-import-order,protected-access,g-import-not-at-top
 
-import distutils as _distutils
+try:
+  import distutils as _distutils
+except ImportError:
+  pass
```

Comment:
Let's completely remove `distutils` here instead of using `try-except` and silence the failure